server/testutil/testnetworking/testnetworking.go

```diff
@@ -33,4 +35,8 @@ func Setup(t *testing.T) {
 		os.WriteFile("/proc/sys/net/ipv4/ip_forward", []byte("1"), 0)
 		require.NoError(t, err, "enable IPv4 forwarding")
 	}
+
+	// Set up default hostNetAllocator
+	err = networking.Configure(context.Background())
+	require.NoError(t, err)
```

Comment:
This change is not directly related but it fixes a panic when running just the SetupAndCleanup test using `--test_filter`.